src/Pages/Login/Signup.js

```diff
@@ -0,0 +1,141 @@
+import React, { Component } from 'react';
+import './Signup.scss';
+
+class Signup extends Component {
+  constructor() {
+    super();
+    this.state = {
+      email: '',
+      password: '',
+      passwordReCheck: '',
+      name: '',
+      phone_number: '',
+    };
+  }
+
+  fetch = e => {
+    e.preventDefault();
+    fetch('http://10.58.2.175:9000/users/signup', {
+      method: 'POST',
+      body: JSON.stringify({
+        email: this.state.email,
+        password: this.state.password,
+        name: this.state.name,
+        phone_number: this.state.phone_number,
+      }),
+    })
+      .then(response => {
+        return response.json();
+      })
```

Comment:
```suggestion
      .then(response => response.json())
```
화살표 함수로 간결하게 표현할 수 있는 곳은 간결하게 표현해주세요!

Reply:
수정했습니다!